main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+
+	"github.com/cloudposse/terraform-provider-utils/internal/provider"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
+	
+)
+
+var (
+	version string = "dev"
```

Comment:
what's the `"dev" version?
Should we change it to prod or something different?

Reply:
That's just used locally in tests.